tested/languages/config.py

```diff
@@ -159,6 +161,29 @@ def limit_output(output: str,
     return '\n'.join(forward_buffer + [ellipsis_str] + backward_buffer[::-1])
 
 
+def trace_to_html(traceback: str, python=False) -> Message:
+    # Escape special characters
+    traceback = html.escape(traceback)
+    # Compile regex for python stacktrace
+    if python:
```

Comment:
I think I would prefer passing `link_regex`  and `link_subs`  as parameters to the function, instead of passing the `python` boolean, so something like this:

```python
def trace_to_html(traceback: str, link_regex=None, link_subs=None) -> Message:
    if not link_regex:
        link_regex = re.compile(r'&lt;code&gt;:([0-9]+)')
    if not link_subs:
        link_subs = r'<a href="#" class="tab-link" data-tab="code" ' \
                    r'data-line="\1">&lt;code&gt;:\1</a>'

    # Escape special characters
    traceback = html.escape(traceback)
    # Add links to
    traceback = link_regex.sub(link_subs, traceback)
    logger.debug(f'<pre><code>{traceback}</code></pre>')
    return ExtendedMessage(
        description=f'<pre><code>{traceback}</code></pre>',
        format="html",
        permission=Permission.STUDENT
    )
```

This way we don't have a Python-specific flag and we can use this function for other languages if they also need a custom regex.